cabra/src/pages/components/Task.tsx

```diff
@@ -1,20 +1,55 @@
 import "twin.macro";
 
+import { getTask, getTasks, updateTask } from "../../api/tasks";
+import { useMutation, useQueryClient } from "react-query";
+
+import CheckBox from "./CheckBox";
 import { ITask } from "../../types/task";
 import { Link } from "react-router-dom";
 import NavigationButton from "./NavigationButton";
 import { formatDateTime } from "../../utils/dates";
 import { routeHelpers } from "../../routes";
+import { useState } from "react";
 
 interface Props {
   task: ITask;
   detailsLink: boolean;
 }
 
 export default function Task({ task, detailsLink }: Props) {
+  const [errorMessage, setErrorMessage] = useState("");
+  const client = useQueryClient();
+  const addTask = useMutation((task: ITask) => updateTask(task.id, task), {
```

Comment:
This name is a bit misleading, can you rename it to sth like `editTask`?

Reply:
Ta nazwa wzięła się z pliku [TaskEdit.tsx](cabra/src/pages/TaskEdit.tsx) bo wydaję mi się że tam też pasowałoby `editTask`.

Reply:
Tak, tam już jest zmienione na developie